packages/FormElement/src/components/Content/Content.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { RefOf } from "@paprika/helpers/lib/customPropTypes";
+import * as styled from "./Content.styles";
+
+const propTypes = {
+  children: PropTypes.oneOfType([PropTypes.func, PropTypes.node]).isRequired,
+  idForLabel: PropTypes.string,
+  refLabel: RefOf(PropTypes.func),
```

Comment:
```suggestion
  refLabel: RefOf(PropTypes.instanceOf(Element)),
```

Reply:
It's not a function, it's the actual dom element, right?

https://stackoverflow.com/a/51127130

Reply:
👍